orbital-mechanics/Lecture12/Lecture12.ipynb

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
<p>Remove the line that says "content about Repeat-Groundtrack-Orbits".</p><p> </p><p>All of the section headings have this as well, so please remove that line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>